build/deploy/prometheus.libsonnet

```diff
@@ -126,7 +126,8 @@ local base = import 'base.libsonnet';
       },
     },
     service: base.Service(metadata, 'prometheus-service') {
-      metadata: {
+      app:: 'prometheus-service',
+      metadata+: {
         name: 'prometheus-service',
         namespace: metadata.namespace,
         annotations: {
```

Comment:
below, under spec, if you do spec+: you'll get the selector for free. 

How did you pick the nodeport value? maybe we can rework base, to check if it's a nodeport and instead of taking a name for the port it takes a nodeport

Reply:
node port value was based on tutorials I found.